wordpress.yaml

```diff
@@ -1,7 +1,152 @@
 ---
 - name: Deploy Wordpress
   hosts: all
+  become: yes
+  vars_files:
+    - /vagrant/vars.yaml
+
   tasks:
+    - name: Update APT
+      apt:
+        update_cache: yes
+        cache_valid_time: 86400
+        upgrade: 'yes'        
+
+    - name: Install packages
+      apt:
+        name: 
+        - python3-mysqldb
+        - nginx
+        - mysql-server
+        - php-fpm
+        - php-mysql
+        - php-curl
+        - php-gd
+        - php-intl
+        - php-mbstring
+        - php-soap
+        - php-xml
+        - php-xmlrpc
+        - php-zip
+          
+    - name: Configure firewall
+      ufw:
+        rule: allow
+        name: "Nginx Full"
+        state: enabled
+    
+    - name:
```

Comment:
+1 for using template. But what is the reason to use template here? :)

Reply:
I thought I could later use some values of the ansible in the conf file. But I will read more about it.

Reply:
that make sence later with variables